components/Profile/pack-list.tsx

```diff
@@ -8,15 +8,14 @@ import { SearchX } from 'lucide-react'
 export interface PackData {
 	type: string
 	id: string
-	featured: boolean
+	popular: boolean
 	name: string
 	language: string
 	description: string
 	tags: string[]
 	likes: string
 	questions: number
-	pending: boolean
-	denied: boolean
+	status: 'pending' | 'approved' | 'resubmit_pending' | 'denied'
```

Comment:
Should probably use the status enum from prisma (don't forget to import)
```suggestion
	status: Status
```